src/types/redis_geo.cc

```diff
@@ -320,7 +320,7 @@ void Geo::scoresOfGeoHashBox(GeoHashBits hash, GeoHashFix52Bits *min, GeoHashFix
  * 'max', appending them into the array of GeoPoint structures 'gparray'.
  * The command returns the number of elements added to the array.
  *
- * Elements which are farest than 'radius' from the specified 'x' and 'y'
+ * Elements which are farther than 'radius' from the specified 'x' and 'y'
```

Comment:
```suggestion
 * Elements which are farthest than 'radius' from the specified 'x' and 'y'
```

Reply:
since there is a `than`, i think it should be farther? (although i am not sure about it)